packages/esm-ward-app/src/beds/bed-share-divider.scss

```
@@ -0,0 +1,18 @@
+@use '@carbon/layout';
+@use '@openmrs/esm-styleguide/src/vars';
+@use '@carbon/type';
+
+.bedDivider {
+  background-color: vars.$ui-02;
+  color: vars.$text-02;
+  padding: layout.$spacing-01;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.bedDividerLine {
+  height: 1px;
+  background-color: vars.$ui-05;
+  width: 30%;
+}
```

```suggestion
@use '@openmrs/esm-styleguide/src/vars' as *;
@use '@carbon/type';

.bedDivider {
  background-color: $ui-02;
  color: $text-02;
  padding: layout.$spacing-01;
  display: flex;
  align-items: center;
  justify-content: space-between;
}

.bedDividerLine {
  height: 1px;
  background-color: $ui-05;
  width: 30%;
}
```

nit: you could also do this